jerry-debugger/jerry-client-ws.py

```diff
@@ -728,6 +757,27 @@ def parse_source(debugger, data):
     else:
         logging.debug("No pending breakpoints")
 
+def printSource(self, line_num):
```

Comment:
Naming style: not camel case but underscores. Thus the name should be: `print_source`.
Additionally, we should not accept self (as we don't know what is it) but instead the debugger itself as the first argument.